docs/model_developers/modules.rst

```diff
@@ -329,3 +329,41 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules form a private python package repository
+--------------------------------------------------------------------------------
+
+Modules v2 are Python packages that can be installed from private Python package repositories, which may require authentication. This section explains the steps to install such modules.
+
+Create a file named ``.netrc`` in the orchestrator's file system, specifically in the "inmanta" directory. 
+Add the following content to the file:
+
+.. code-block:: text
+
+  machine <url of the private repository>
+  login token
+  password <the token to access the private repository>
+
+In the ``project.yml`` file of your project, ensure that the ``use_config_file`` option under the ``pip`` section is set to ``True``.
+
+Create a file named ``pip.conf`` and add the following content to it:
+
+.. code-block:: text
+
+  [global]
+  timeout = 60
+  index-url = <url of the repository >
+
+Set the ``PIP_CONFIG_FILE`` environment variable to the path of the ``pip.conf`` file. You can do this by running the following command:
+
+.. code-block:: bash
+
+  $ export PIP_CONFIG_FILE=<path/to/pip.conf>
+
+Run the following command to install the modules:
+
+.. code-block:: bash
+
+  $ inmanta project install
+
+By following these steps, you will be able to configure your project to install modules from a private Python package repository.
```

Comment:
The compiles service will execute these, so no need to mention them here. Unless we want to have a separate section for compiles on the CLI, but I am not sure whether we need that.

Reply:
I'll remove that part